utils/src/wizard/ConfirmationStep.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import * as types from "../../index";
+import { DialogResponses } from "../DialogResponses";
+import { AzureWizardPromptStep } from "./AzureWizardPromptStep";
+
+export class ConfirmationStep extends AzureWizardPromptStep<types.IActionContext> {
+
+    public constructor(private readonly message: string) { super(); }
+
+    public async prompt(context: types.IActionContext): Promise<void> {
+        await context.ui.showWarningMessage(this.message, { modal: true, stepName: 'confirmation' }, DialogResponses.yes, DialogResponses.no);
```

Comment:
Now that I'm looking at it, I'm not sure how this in particular would work; the `yes` / `no` response isn't carried anywhere. It's different from `DeleteConfirmationStep` which would simply pass if yes, and otherwise throw `UserCancelledError`.

In retrospect maybe we should just do the `DeleteConfirmationStep`.

Reply:
Good catch. It's too early in the morning 🤣, I mindlessly added this step after reading your feedback without realizing that it would never work.

Reply:
I can't even use the "too early in the morning" excuse, it's after noon!